www/html/ci/application/views/base.html

```diff
@@ -4,72 +4,76 @@
 <html lang="en">
 
 <head>
-    <meta content="text/html; charset=utf-8" http-equiv="Content-Type" />
-    <title>Ken Torii &copy;</title>
-    <meta content="width=device-width, initial-scale=1" name="viewport" />
-    <link href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet" />
-    <link href="{{ base_url }}font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
-    <link href="{{ base_url }}css/index2.css" rel="stylesheet" />
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
-    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
-    <script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
-    <script type="text/javascript" src="{{ base_url }}js/webJS.js"></script>
+    {% block head %}{% endblock %}
```

Comment:
Can you explain this change? I don't think I should see the header change in base template. That's the whole point of having a template file.